src/researches/findKeywordInFirstParagraph.js

```diff
@@ -16,13 +19,44 @@ import { isEmpty } from "lodash-es";
  * @returns {boolean} True if the text consists only of images, false otherwise.
  */
 function paragraphConsistsOfImagesOnly( text ) {
```

Comment:
Since this function also checks for anchor tags and empty divisions as well, I would update this function's name. Perhaps `paragraphDoesNotContainText` ?